ibu-imager/ops/ops.go

```diff
@@ -209,3 +211,15 @@ func (o *ops) RunRecert(recertContainerImage, authFile, recertConfigFile string,
 
 	return nil
 }
+
+func (o *ops) ExtractTarWithSELinux(srcPath, destPath string) error {
+	_, err := o.hostCommandsExecutor.Execute(
+		"tar", "xzf", srcPath, "-C", destPath, "--selinux",
+	)
+	return err
+}
+
+func (o *ops) RemountSysroot() error {
```

Comment:
I am not sure.
but `hostCommandExecutor` is `ChrootExecutor` here. the command executed is `Executing /usr/bin/env with args [-- bash -c mount /sysroot -o remount,rw]`

Reply:
SSH to the host after prep is done and check the /sysroot mountpoint to see if it's rw or ro.

Reply:
I think it should be ok, that it will be ro on the node. You could also rsh into the LCA manager container to check there, but that's less of a concern, and would likely be dependent on whether the command is being launched in a go thread or in the main thread.

Reply:
```
sh-5.1# cat /proc/mounts | grep "/sysroot"
/dev/nvme0n1p4 /sysroot xfs ro,seclabel,relatime,attr2,inode64,logbufs=8,logbsize=32k,prjquota 0 0
/dev/nvme0n1p4 /sysroot/ostree/deploy/rhcos/var xfs rw,seclabel,relatime,attr2,inode64,logbufs=8,logbsize=32k,prjquota 0 0
```
It looks like it is read only